docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.7"
```

Comment:
Depending on the docker-compose version available in travis, this is probably a too high version of compose file to ask for, see https://docs.docker.com/compose/compose-file/compose-versioning/ and https://docs.travis-ci.com/user/docker/#using-docker-compose

Reply:
Compose file with version 3.7 supports docker-engine 18.06.0+.
We are now running on the Xenial build dist of Travis.
This build dist has docker-engine in version 18.06.0-ce and docker-compose in version 1.23.1.

The version 3.7 should be the most up-to-date version we can use for now.